themes/default/content/docs/intro/concepts/resources/options/ignorechanges.md

```diff
@@ -91,6 +91,37 @@ One reason you would use the `ignoreChanges` option is to ignore changes in prop
 
 In addition to passing simple property names, nested properties can also be supplied to ignore changes to a more targeted nested part of the resource's inputs. Here are examples of legal paths that can be passed to specify nested properties of objects and arrays, as well as to escape object keys that contain special characters:
 
+{{< chooser language "javascript,typescript,python,go,csharp,java,yaml" >}}
+
+{{% choosable language python %}}
+
+{{% notes "info" %}}
+The property names passed to `ignoreChanges` should always be the "camelCase" version of the property name, as used in the core Pulumi resource model.
```

Comment:
why is camel case in quotes here?

Reply:
It's how I found it! I think it makes sense since camelCase isn't a real English word, or at least a very niche one. Happy to remove